src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -246,7 +246,7 @@ private Map<String, Object> getMapFromJsonNodeForStreamingIngest(JsonNode node)
     while (columnNames.hasNext()) {
       String columnName = columnNames.next();
       JsonNode columnNode = node.get(columnName);
-      columnName = formatColumnName(columnName);
+      // column should be passed without processing
```

Comment:
nit: please remove, people who read the code won't know the history that column needs processing :)